scripts/pre-commit.sh

```diff
@@ -93,4 +99,13 @@ DOCKER_PUSH=false ./scripts/deploy-federation.sh quay.io/kubernetes-multicluster
 echo "Running go e2e tests"
 run-e2e-tests
 
+echo "Deleting federation-v2"
+./scripts/delete-federation.sh
+
+echo "Deploying namespaced federation-v2"
+FEDERATION_NAMESPACE=foo NAMESPACED=y DOCKER_PUSH=false ./scripts/deploy-federation.sh quay.io/kubernetes-multicluster/federation-v2:e2e
+
+echo "Running go e2e tests"
+run-namespaced-e2e-tests
+
```

Comment:
@marun @font do we also need to delete the federation controller as well at here as in L102-L103?

Reply:
The cluster-scoped control plane is deleted to ensure it won't conflict with the namespaced control plane.  Cleanup of the namespaced control plane would only add overhead to the ci job, since vm for the job is thrown away at completion.